Assets/PatchKit Patcher/Scripts/MultipleInstanceException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace PatchKit.Unity.Patcher
+{
+    [Serializable]
+    internal class MultipleInstancesException: Exception
```

Comment:
I see multiple class declarations in MultipleInstanceException and MultipleInstance**s**Exception.